src/docs/product/security-policy-reporting.mdx

```diff
@@ -62,7 +62,7 @@ In addition to the `sentry_key` parameter, you may also pass the following withi
 
 `sentry_environment`
 
-: The environment name (e.g. production). The environment name can't contain newlines, spaces, or forward slashes, can't be the string "None", or exceed 64 characters. Also, environments are case sensitive. 
+: The environment name (e.g. production). The environment name can't contain newlines, spaces, or forward slashes, can't be the string "None", or exceed 64 characters. Also, environments are case sensitive.
```

Comment:
```suggestion
: The environment name (for example, production). 
The environment name is case sensitive, can't contain new lines, spaces, or forward slashes. It can't be the string "None" or exceed 64 characters. 
```